flavors/vpp/vpp_flavor.go

```diff
@@ -28,12 +28,14 @@ func (f *Flavor) Inject() error {
 
 	f.Base.Inject()
 
-	f.GoVPP.StatusCheck = &f.Base.Generic.StatusCheck
-	f.GoVPP.LogFactory = &f.Base.Generic.Logrus
-	f.VPP.ServiceLabel = &f.Base.Generic.ServiceLabel
+	f.GoVPP.Deps.PluginInfraDeps = *f.Base.FlavorLocal.InfraDeps("GOVPP")
+	f.VPP.Deps.PluginInfraDeps = *f.Base.FlavorLocal.InfraDeps("default-plugins")
+	f.VPP.Transport = &f.Base.ETCDDataSync
```

Comment:
may be better name since transport & watch seems to be simmilar.

Maybe: Watch, Publish.